cmd/tsconnect/build-pkg.go

```diff
@@ -41,4 +47,33 @@ func runBuildPkg() {
 		log.Fatalf("Type generation failed: %v", err)
 	}
 
+	if err := updateVersion(); err != nil {
+		log.Fatalf("Cannot update version: %v", err)
+	}
+
+	log.Printf("Built package version %s", version.Long)
+}
+
+func updateVersion() error {
+	packageJSONBytes, err := os.ReadFile("pkg-package.json")
```

Comment:
Should we call this `package.json.tmpl` or something to imply it's not the final file? `pkg-package` feels like "package package" which doesn't tell me much.

Reply:
Yeah, `package.json.tmpl` is better.